draft-dekater-scion-controlplane.md

```diff
@@ -1139,6 +1139,7 @@ Depending on the selection criteria, it may be necessary to keep more candidate
 
 - The *propagation interval* SHOULD be at least "5" (seconds) for intra-ISD beaconing and at least "60" (seconds) for core beaconing.
 
+Note that during bootstrapping and if the AS obtains a PCB containing a previously unknown path, the AS SHOULD forward the PCB more frequently, to ensure quick connectivity establishment.
```

Comment:
I side with Matthias regarding the phrase "bootstrapping". This calls for a definition and such is hard to come by. Do we need to mention that at all?

Regarding "The AS SHOULD". We just said that it "SHOULD not" in the paragraph above. So can't possibly express it like that. This may be one case where "MAY" is approriate, assuming we specify under which condition (since that derogates from the previous paragraph.

Regarding when it MAY... I think that "previously unknown" segments, or "previously unreachable destinations" could qualify.

Reply:
"Bootstrapping" was a quick & dirty word, I'll replace it with a list of conditions when the propagation interval MAY not be respected.

Reply:
(fee free to resolve if you like my text, or to propose an alternative)